docs/en/operation/Metrics-System.md

```diff
@@ -135,190 +136,84 @@ Tags can be used to further filter or aggregate on various characteristics.
 Workers and clients send metrics data to the Alluxio master through heartbeats.
 The interval is defined by property `alluxio.master.worker.heartbeat.interval` and `alluxio.user.metrics.heartbeat.interval` respectively.
 
-Each client will be assigned an application id. 
-All the metrics sent by this client contain the client application id information. 
-By default, this will be in the form of 'app-[random number]'. 
-This value can be configured through the property `alluxio.user.app.id`, 
-so multiple clients can be combined into a logical application.
-
-#### Alluxio Cluster Information
-
-| Metric Name | Description |
-|-------------------------|-----------------------------------------------|
-| Master.Workers | Total number of active Alluxio workers in this cluster |
-
-#### Alluxio Storage Capacity
-
-| Metric Name | Description |
-|--------------------------------|--------------------------------------------------------------------------|
-| Master.CapacityTotal | Total capacity of the Alluxio file system in bytes |
-| Master.CapacityTotalTier<TIER> | Total capacity in tier <TIER> of the Alluxio file system in bytes |
-| Master.CapacityUsed | Used capacity of the file system in bytes |
-| Master.CapacityUsedTier<TIER> | Used capacity in tier <TIER> of the Alluxio file system in bytes |
-| Master.CapacityFree | Free capacity of the Alluxio file system in bytes |
-| Master.CapacityFreeTier<TIER> | Free capacity in tier <TIER> of the Alluxio file system in bytes |
-
-#### Under Storage Capacity
-
-| Metric Name | Description |
-|-------------------------|--------------------------------------------------|
-| Master.UfsCapacityTotal | Total capacity of the under file system in bytes |
-| Master.UfsCapacityUsed | Used capacity of the under file system in bytes |
-| Master.UfsCapacityFree | Free capacity of the under file system in bytes |
+<table class="table table-striped">
+<tr><th>Name</th><th>Type</th><th>Description</th></tr>
+{% for item in site.data.table.cluster-metrics %}
+  <tr>
+    <td><a class="anchor" name="{{ item.metricName }}"></a> {{ item.metricName }}</td>
+    <td>{{ item.metricType }}</td>
+    <td>{{ site.data.table.en.cluster-metrics[item.metricName] }}</td>
+  </tr>
+{% endfor %}
+</table>
```

Comment:
This page will be quite large. Maybe we can add a new page to the "Reference" dropdown?

Reply:
Moved to the Reference as Metrics-list.md